ladybug_geometry/geometry2d/polygon.py

```diff
@@ -313,10 +313,62 @@ def is_valid(self):
         """
         return not self.area == 0
 
+    def is_equivalent(self, other, tolerance):
+        """ Boolean noting equivalence (based on point tolerance) between this polygon and
+        another polygon. The order of the polygon vertices do not have to start from the
+        same vertex for equivalence to be true, but must be in the same counterclockwise
+        or clockwise order.
+
+        Args:
+            other: Polygon2D for comparison.
+            tolerance: float representing point tolerance.
+
+        Returns:
+            True if equivalent else False
+        """
+
+        # Check number of points
+        if len(self.vertices) != len(other.vertices):
+            return False
+
+        vertices = list(self.vertices)
```

Comment:
The casting to list here seems unnecessary. I don't see you mutating the `vertices` variable at any point in the rest of your code.